src/utils/create-utils.ts

```diff
@@ -151,3 +159,75 @@ export const createComponentDetectionQuery = async (
 
   return cdq.status.componentDetected;
 };
+
+/**
+ * Create SPIAccessTokenBinding CR
+ *
+ * @param url the URL of the git repository
+ * @param namespace namespace to create the binding
+ * @returns Returns created SPIAccessTokenBinding resource
+ */
+export const createAccessTokenBinding = async (url: string, namespace: string) => {
+  const id = uid();
+  const requestData = {
+    apiVersion: `${SPIAccessTokenBindingModel.apiGroup}/${SPIAccessTokenBindingModel.apiVersion}`,
+    kind: SPIAccessTokenBindingModel.kind,
+    metadata: {
+      name: `hac-dev-import-${id}`,
```

Comment:
We don't want to introduce `hac-dev` as a prefix. 
Maybe use `appstudio` instead unless there's another name given to appstudio that's more user facing?

Reply:
Updated to use the `appstudio` prefix.